MockKit/MockCGMManager.swift

```diff
@@ -254,7 +254,16 @@ extension MockCGMLifecycleProgress: RawRepresentable {
 public final class MockCGMManager: TestingCGMManager {
     
     public static let managerIdentifier = "MockCGMManager"
+
+    public var managerIdentifier: String {
+        return MockCGMManager.managerIdentifier
```

Comment:
[nit] If the static is no longer necessary, why not just collapse into:
```
public var managerIdentifier: String { "MockCGMManager" }
```
and remove the static?

Reply:
Here and below for `localizedTitle`. Also in `MockPumpManager.swift`.

Reply:
@darinkrauss There is still code in the Mock devices that utilize the static, and that's ok, as they are not re-using types for multiple DeviceManager instances.